### ISSUE TITLE
consider adding miniprot for spliced alignment

### ISSUE THREAD

**Issue by hyphaltip** [nextgenusfs/funannotate#814]
It may be worth trying [miniprot](https://github.com/lh3/miniprot) for spliced alignments as an alternative path for generating these exon support results.

Though in my tests I found it didn't always get the full alignment splicing boundaries so may need some additional testing.  But this combines the search (diamond/tblastn) + spliced align (exonerate) steps and maybe potentially faster?

**Comment by nextgenusfs** [comment]
Yes worth adding -- but probably just wait for funannotate v2 -- a slew of other changes are waiting as well.
